wallet/db.c

```diff
@@ -950,10 +950,10 @@ struct db *db_setup(const tal_t *ctx, struct lightningd *ld,
 	db->report_changes_fn = plugin_hook_db_sync;
 
 	db_begin_transaction(db);
+	db->data_version = db_data_version_get(db);
```

Comment:
Got confused about why this works. The DB version and the data version are orthogonal to each other :-)